client/src/components/Overview/ImageCarousel.jsx

```diff
@@ -6,11 +6,11 @@ import React from 'react';
 // }
 
 const ImageCarousel = (props) => {
-  {if (props.currentStyle.photos) {
-    {console.log('FOR THUMBIES', props.currentStyle.photos)}
+  {if (props.productImageCarousel !== []) {
+    {console.log('ooohhhh', props)}
```

Comment:
left a console.log here

Reply:
dangit i thought i got em all, thanks!! ill take it out and push again in a sec (: